#The mesage to be sent to the arduino for controlling the motorss
Header header
#front left motor
int8 speed1
int8 dir1
int8 break1

#back left motor
int8 speed2
int8 dir2
int8 break2

#front right motor
int8 speed3
int8 dir3
int8 break3

#back right motor
int8 speed4
int8 dir4
int8 break4


int8 globalEN

